src/bucket/bucket.html

```diff
@@ -45,4 +45,13 @@
     <uploadform config="$ctrl.uploadConfig" folder="$ctrl.currentFolder">
     </uploadform>
   </div>
+
+  <!-- dropzone -->
+  <div class="s3c-bottom-section"
+    ng-if='$ctrl.uploadConfig.url'>
```

Comment:
Why is this `ng-if` newcessary here?

Reply:
Yeah, this `ng-if` is necessary to allow the dropzone element to be initialized after `uploadConfig` is set and `url` is not null.

I am using the `uploadConfig.url` in the dropzone config. Before, the dropzone element would initialize while `uploadConfig` was undefined, and the url would not be set properly in the dropzone config (causing the POST to fail).

Reply:
Good point, this has been changed so `uploadConfig` is tested prior to initializing the dropzone element.